modules/rhcos-enabling-multipath-day-2.adoc

```diff
@@ -11,7 +11,7 @@
 ====
 Enabling multipathing during installation is supported and recommended for nodes provisioned in {product-title} 4.8 or higher. In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time. For more information about enabling multipathing during installation time, see "Enabling multipathing with kernel arguments on RHCOS" in the _Installing on bare metal_ documentation.
 ====
-
+//Should the IMPORTANT admonition also include "IBM Power"?
 [IMPORTANT]
 ====
 On IBM Z and LinuxONE, you can enable multipathing only if you configured your cluster for it during installation. For more information, see "Installing {op-system} and starting the {product-title} bootstrap process" in _Installing a cluster with z/VM on IBM Z and LinuxONE_.
```

Comment:
@ktania46 or @SNiemann15 - Should this note be updated to say, "On IBM Z and LinuxONE _or IBM Power Systems_, you can enable multipathing only if you configured your cluster for it during installation"?